var/spack/repos/builtin/packages/py-docutils/package.py

```diff
@@ -29,7 +29,10 @@ class PyDocutils(PythonPackage):
     depends_on('py-setuptools',       when='@0.15:',         type='build')
     depends_on('python@2.7:2.8,3.5:', when='@0.16:',         type=('build', 'run'))
     depends_on('python@2.6:2.8,3.3:', when='@0.14:0.15',     type=('build', 'run'))
-    depends_on('python@2.4:3',        when='@:0.13',     type=('build', 'run'))
+    depends_on('python@2.4:3',        when='@0.10:0.13',     type=('build', 'run'))
+    depends_on('python@2.2.1:3',      when='@0.6:0.9',       type=('build', 'run'))
```

Comment:
Why do these cap at `:3`?  That isn't excluding anything.

Reply:
It excludes future Python 4 releases. But yeah, the package doesn't explicitly state that it doesn't support Python 4, so we should probably remove the upper limit. Will do.